.github/actions/cache/action.yaml

```diff
@@ -16,13 +16,13 @@ runs:
         set -e
 
         # Should be the same command as the one in tools/static/castor.php
-        cache_dirname_linux_amd64=$(tests/bin/compile-get-cache-key phar-location-is-not-used-in-cache-key --os=linux --arch=x86_64 --php-extensions=mbstring,phar,posix,tokenizer,pcntl)
+        cache_dirname_linux_amd64=$(tests/bin/compile-get-cache-key phar-location-is-not-used-in-cache-key --os=linux --arch=x86_64 --php-extensions=mbstring,phar,posix,tokenizer,pcntl,curl)
```

Comment:
curl is a good default. More over, without it, we'll miss "update instruction" because castor cannot fetch github releases (HTTPS is not supported with low-cost http client"